deployment/terraform/metrics.go

```diff
@@ -124,19 +124,19 @@ func (t *Terraform) setupMetrics(extAgent *ssh.ExtAgent) error {
 		mmTargets = append(mmTargets, fmt.Sprintf("%s:8067", host))
 		nodeTargets = append(nodeTargets, fmt.Sprintf("%s:9100", host))
 		netpeekTargets = append(netpeekTargets, fmt.Sprintf("%s:9045", host))
-		hosts += fmt.Sprintf("%s %s\n", val.GetConnectionIP(), host)
```

Comment:
I had a feeling that things might go wrong when we added the GetConnectionIP method. Very nice work on this!